src/Howerpoint.hs

```diff
@@ -120,49 +138,61 @@ padding :: Int -> Int -> (Int, Int)
   where showableContent = filter (not . isHiddenCode) content
         (topPadding, bottomPadding) = padding (height ps) (length showableContent)
         lineOfStars = take (width ps) $ repeat '*'
-        emptyLine = '*' : (repeatN ((width ps) -2) ' ') ++ "*"
+        emptyLine = '*' : (replicate ((width ps) -2) ' ') ++ "*"
         (leftCodePadding, _) = padding (width ps) (maxCodeLength showableContent - 3)
         trimCode :: String -> String
         trimCode = drop 3
         centerLine :: String -> String
         centerLine lineContent =
-          let (colorizedContent, len) = if isCode lineContent then colorize (trimCode lineContent) else colorize lineContent
-              (leftPadding, rightPadding) = if isCode lineContent then (leftCodePadding, (width ps) - leftCodePadding - len) else padding (width ps) len
+          let (colorizedContent, len) =
+                if isCode lineContent
+                    then colorize (trimCode lineContent)
+                    else colorize lineContent
+              (leftPadding, rightPadding) =
+                if isCode lineContent
+                  then (leftCodePadding, (width ps) - leftCodePadding - len)
+                  else padding (width ps) len
           in '*' :
-             (repeatN (leftPadding - 1) ' ') ++
+             (replicate (leftPadding - 1) ' ') ++
              colorizedContent ++
-             (repeatN (rightPadding - 1) ' ') ++
+             (replicate (rightPadding - 1) ' ') ++
              "*"
 
 -- foreground colors 30-37, background colors 40-47
 color :: Char -> Maybe String
-color x = fmap (\c -> "\x1b[" ++ show (if isLower x then c + 30 else c + 40) ++ "m" )
-  (case toLower x of  'k' -> Just 0 -- Black
-                      'r' -> Just 1 -- Red
-                      'g' -> Just 2 -- Green
-                      'y' -> Just 3 -- Yellow
-                      'b' -> Just 4 -- Blue
-                      'm' -> Just 5 -- Magenta
-                      'c' -> Just 6 -- Cyan
-                      'w' -> Just 7 -- White
-                      _   -> Nothing -- No coloring
-  )
-
+color x =
+  fmap (\c -> "\x1b[" ++ show (if isLower x then c + 30 else c + 40) ++ "m") charColor
+  where
+    charColor = case toLower x of
+      'k' -> Just 0 -- Black
+      'r' -> Just 1 -- Red
+      'g' -> Just 2 -- Green
+      'y' -> Just 3 -- Yellow
+      'b' -> Just 4 -- Blue
+      'm' -> Just 5 -- Magenta
+      'c' -> Just 6 -- Cyan
+      'w' -> Just 7 -- White
+      _   -> Nothing -- No coloring
+
+colorReset :: String
 colorReset = "\x1b[0m"
 
 -- it also takes care of escaping to ;\\k' will be just '\k' and not black color
 colorize :: String -> (String, Int)
-colorize line = if elem '\\' line then colorize' line [] (length line) else (line, length line)
+colorize line =
+  if elem '\\' line
+    then colorize' line [] (length line)
+    else (line, length line)
```

Comment:
I find `if/then/else` broken onto 3 lines easier to read, unless it's a really short if/then/else. This is personal preference.